extensions/mongodb-client/runtime/pom.xml

```diff
@@ -28,6 +28,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-devservices</artifactId>
+        </dependency>
```

Comment:
Just to be totally clear, this does not result in the inclusion of any additional dependencies in prod mode, right?

Reply:
Yes, the runtime module (`quarkus-devservices`) only depends on `quarkus-arc`.

Reply:
Great, thanks for clarifying!